Description: Allow routes 2.0
 The python-routes 2.0 from Debian is perfectly working, the issue with 2.0
 is only the PyPi release, so we patch requirements.txt to allow it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-10-02

Index: heat/requirements.txt
===================================================================
--- heat.orig/requirements.txt	2015-02-07 12:35:28.000000000 +0100
+++ heat/requirements.txt	2015-02-07 12:35:32.000000000 +0100
@@ -34,7 +34,7 @@
 PyYAML>=3.1.0
 qpid-python
 requests>=1.2.1,!=2.4.0
-Routes>=1.12.3,!=2.0
+Routes>=1.12.3
 six>=1.7.0
 SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
 sqlalchemy-migrate==0.9.1
